index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+
+  <nav id="navbar">
+    <header>JS Documentation</header>
+    <a href="#Introduction" class="nav-link">Introduction</a>
+    <a href="#What_you_should_already_know" class="nav-link">What you should already know</a>
+    <a href="#JavaScript_and_Java" class="nav-link">JavaScript and Java</a>
+    <a href="#Hello_world" class="nav-link">Hello world</a>
+    <a href="#Declaring_Variables" class="nav-link">Declaring Variables</a>
+    <a href="#Variables_Scope" class="nav-link">Variables Scope</a>
+  </nav>
+  <main id="main-doc">
+    <section class="main-section" id="Introduction">
+      <header>Introduction</header>
+      <p>JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language.
+        Inside a host environment (for example, a web browser),
+        JavaScript can be connected to the objects of its environment to provide programmatic control over them.</p>
+      <p>JavaScript contains a standard library of objects, such as Array, Date, and Math, and a core set of language
+        elements such as operators, control structures, and statements. Core
+        JavaScript can be extended for a variety of purposes by supplementing it with additional objects; for example:
+      </p>
+      <ul>
+        <li>Client-side JavaScript extends the core language by supplying objects to control a browser and its Document
+          Object Model (DOM). For example, client-side extensions allow an application to place
+          elements on an HTML form and respond to user events such as mouse clicks, form input, and page navigation.
+        </li>
+        <li>Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on a
+          server. For example, server-side extensions allow an application to communicate with a database, provide
+          continuity of
+          information from one invocation to another of the application, or perform file manipulations on a server.</li>
+      </ul>
+    </section>
+
+    <section class="main-section" id="What_you_should_already_know">
+      <header>What you should already know</header>
```

Comment:
I think it would be nice if you use the html heading tags for your headings. You can use the header tag to group together elements that you want to be part of you heading section.